docs/cli/index.mdx

```diff
@@ -138,39 +137,49 @@ Enable auto-completion using the following commands.
 
 ### zsh auto-completion
 
-<!-- TODO: add more information about zsh to make comparable to bash section -->
+1. Add the following line to your `~/.zshrc` file:
```

Comment:
The pedant in me:

```suggestion
1. Add the following line to your `~/.zshrc` configuration file:
```

Reply:
This is may be a little pedantic in turn, but it's actually a script, not a configuration file—it can run arbitrary code when `zsh` is launched. The `zsh` manpage calls it a "shell startup file", the `bash` manpage calls it an "initalization file`; I think I'll go with "startup script" if that works for you?

Reply:
If memory serves, rc means "run commands" so have at it. Generally, I have described it as a configuration file that customizes the launch of a shell. Adding some kind of modifier explains what you're changing to any people who don't know _why_ they're editing the file. I like shell initialization slightly more than just initialization.